server.js

```diff
@@ -18,7 +18,79 @@ app.use(express.json());
 
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+const navigation = {
+    guide: "Routes for this API",
+    Routes: [
+      {
+      "/thoughts": "Happy thoughts API"
+      },
+    ],
+  };
+  res.send(navigation);
+});
+
+const ThoughtSchema = new mongoose.Schema({
+  message: {
+    type: String,
+    required: true,
+    unique: true,
+    minlength: 2,
+    maxlength:30,
+    trim: true,
+  },
+  hearts: {
+    type: Number,
+    default: 0,
+  },
+  createdAt: {
+    type: Date,
+    default: () => new Date()
+  }
+});
+
+const Thought = mongoose.model("Thought", ThoughtSchema);
+
+app.post("/thoughts", async (req, res) => {
+  const { message } = req.body;
+    try {
+    const newThought = await new 
+    Thought({message}).save()
+    res.status(201).json({
+      success: true, 
+      response: newThought})
+    } catch(err) {
+      res.status(400).json({
+      message: "Could not post this thought",
+      response: err.errors,
+      success: false,
+    })}
+});
+
+app.get("/thoughts", async (req, res) => {
+    try {
+    const thoughts = await new 
+    Thought.find().limit(20).sort({ createdAt: "desc" })
+    res.status(200).json({
+    success: true,
+    response: thoughts
+    })
+  } catch (err) {
+    res.status(400).json({
+      message: "Could not find this post",
+      response: err.errors,
+      success: false,
+    });
+  }
+});
+
+app.patch("/members/:id/like", async (req, res) => {
```

Comment:
Don't forget to double check the id-name in your happy thought project when you link together the project :)